bodhi/server/templates/override.html

```diff
@@ -138,6 +138,7 @@ <h2 class="pull-left">New Override</span>
 
     <input type="hidden" name="csrf_token" value="${request.session.get_csrf_token()}"/>
 
+    %if request.user and ('provenpackager' or 'packager' in request.user.groups):
```

Comment:
We should not hardcode the group names, as [they are not hardcoded on the view that checks whether the acl is valid or not](https://github.com/fedora-infra/bodhi/blob/2.3.1/bodhi/server/security.py#L39-L42):

```
    groups = request.registry.settings['mandatory_packager_groups'].split()
    return [
        (Allow, 'group:' + group, ALL_PERMISSIONS) for group in groups
    ] + [DENY_ALL]
```